Description: default to act like fsck
 Inspect arguments, if we are not called as btrfs, then assume we are
 called to act like fsck.
Author: Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>
Bug-Debian: http://bugs.debian.org/712078

--- a/btrfs.c
+++ b/btrfs.c
@@ -271,7 +271,7 @@
 	else
 		bname = argv[0];
 
-	if (!strcmp(bname, "btrfsck")) {
+	if (strcmp(bname, "btrfs") != 0) {
 		argv[0] = "check";
 	} else {
 		argc--;
